docs/release-notes/release-notes-0.6.0.md

```diff
@@ -0,0 +1,149 @@
+- Restyle the Markets page. The new style is responsive, with different layouts for various screen sizes. (<https://github.com/decred/dcrdex/pull/1825> <!-- related: <https://github.com/decred/dcrdex/commit/c0c018993b8c788291969e17cc7dee14a3c522c5>, <https://github.com/decred/dcrdex/commit/0c0a962b9bc4f5b52e6d5393d20802dcc50edb9c>, <https://github.com/decred/dcrdex/commit/9547a5f24e941dfaaf8131c9614fc07ee727ba97>, <https://github.com/decred/dcrdex/commit/8c21d958ad3c548595a25697b694c89b58b3ceff>, <https://github.com/decred/dcrdex/commit/a47bfb14900a29cd6b5910dc69bfd086686145e9>, <https://github.com/decred/dcrdex/commit/1b02525188ead12cd444c6063a3db1e5f374456f>, <https://github.com/decred/dcrdex/commit/7bd33117e54931c7f298bee0c2465d4e0207ef8e>, <https://github.com/decred/dcrdex/commit/399134f19e5838844f3f3e7fa2c60441eefbe3c7> -->
+- Restyle the Wallets page.  The new style is also responsive. (<https://github.com/decred/dcrdex/pull/1700> <!-- related: <https://github.com/decred/dcrdex/commit/4df561131bea62e185c946879ed9bb77790aee5c>, <https://github.com/decred/dcrdex/commit/4e2100d6a9124114e24570fda079f02520e1eba3>, <https://github.com/decred/dcrdex/commit/a891a7793e5fcc125ea7b952d486e69159d4e2b5>, <https://github.com/decred/dcrdex/commit/aa55b045090c4e99d895abb33dbaf1870ef988d6>, <https://github.com/decred/dcrdex/commit/1b50969f631e69bad17eba0963f4cc55bc7c4abf>, <https://github.com/decred/dcrdex/commit/2421de9dc04ac8fea89fea0167d38281c5eccc11>, <https://github.com/decred/dcrdex/commit/a40f58e7c6438eb4ab8ff869a674edc239db6723>, <https://github.com/decred/dcrdex/commit/424d6d223ea73344000b6f76fb5175c99b0edcbf> -->
```

Comment:
These two missing the closing parenthesis on the right.
![image](https://user-images.githubusercontent.com/34998433/230599525-68f665af-cb70-4d53-8929-878cc80df7af.png)